### PR TITLE
chore(build): attempt to add semantic-release

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

File: config/dist/webpack-assets.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "vendor": {
```
Comment (joshuawilson) [inline config/dist/webpack-assets.json]: did you really want this generated code committed? or should it go in the gitignore?

Comment (pmuir) [inline config/dist/webpack-assets.json]: it's needs to be ignore :-)

**Comment by pmuir** [comment]
DO NOT MERGE

**Comment by pmuir** [comment]
[test]

**Comment by pmuir** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 2d9b0d6ff5bde420c5fdb62d299ea29943194c5e
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Pull request closed without merging**